github/table_github_organization_member.go

```diff
@@ -0,0 +1,150 @@
+		{Name: "followers_url", Type: proto.ColumnType_STRING, Description: "The URL of the user's followers."},
+		{Name: "following_url", Type: proto.ColumnType_STRING, Description: "The URL of the user's following."},
+		{Name: "gists_url", Type: proto.ColumnType_STRING, Description: "The URL of the user's gists."},
+		{Name: "gravatar_id", Type: proto.ColumnType_STRING, Description: "The user's gravatar ID."},
+		{Name: "html_url", Type: proto.ColumnType_STRING, Description: "The GitHub page for the user."},
+		{Name: "node_id", Type: proto.ColumnType_STRING, Description: "The node ID of the user."},
+		{Name: "organizations_url", Type: proto.ColumnType_STRING, Description: "The URL of the user's organizations."},
+		{Name: "received_events_url", Type: proto.ColumnType_STRING, Description: "The URL of the user's received events."},
+		{Name: "repos_url", Type: proto.ColumnType_STRING, Description: "The URL of the user's repos."},
+		{Name: "site_admin", Type: proto.ColumnType_BOOL, Description: "If true, user is an administrator."},
+		{Name: "starred_url", Type: proto.ColumnType_STRING, Description: "The URL of the user's stars."},
+		{Name: "subscriptions_url", Type: proto.ColumnType_STRING, Description: "The URL of the user's subscriptions."},
+		{Name: "type", Type: proto.ColumnType_STRING, Description: "The type of account."},
+		{Name: "url", Type: proto.ColumnType_STRING, Description: "The URL of the user."},
+
+		{Name: "role", Type: proto.ColumnType_STRING, Description: "The organization member's role.", Hydrate: tableGitHubOrganizationMemberGet},
+		{Name: "state", Type: proto.ColumnType_STRING, Description: "The membership state.", Hydrate: tableGitHubOrganizationMemberGet},
+	}
+}
+
+func tableGitHubOrganizationMember() *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_organization_member",
+		Description: "GitHub members for a given organization. GitHub Users are user accounts in GitHub.",
+		List: &plugin.ListConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "organization", Require: plugin.Required},
+				{Name: "role", Require: plugin.Optional},
+			},
+			Hydrate:           tableGitHubOrganizationMemberList,
+			ShouldIgnoreError: isNotFoundError([]string{"404"}),
+		},
+		Columns: gitHubOrganizationMemberColumns(),
+	}
+}
+
+//// LIST FUNCTION
+
+func tableGitHubOrganizationMemberList(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	client := connect(ctx, d)
+	opt := &github.ListMembersOptions{
+		ListOptions: github.ListOptions{PerPage: 100},
+		Role:        "all",
+	}
+	quals := d.KeyColumnQuals
+	org := quals["organization"].GetStringValue()
+
+	// Additional filters
+	if quals["role"] != nil {
+		opt.Role = quals["role"].GetStringValue()
+	}
+	type ListPageResponse struct {
+		members []*github.User
+		resp    *github.Response
+	}
+	limit := d.QueryContext.Limit
+	if limit != nil {
+		if *limit < int64(opt.PerPage) {
+			opt.PerPage = int(*limit)
+		}
```

Comment:
@LalitTurbot the query works as expected because the `ListOptions` is an inherited member in `ListMembersOptions` 

```sql
> .cache clear
> select organization,login from github_organization_member where organization = 'new-testing-org' limit 2
+-----------------+-------------+
| organization    | login       |
+-----------------+-------------+
| new-testing-org | LalitTurbot |
| new-testing-org | karanpopat  |
+-----------------+-------------+
```